### ISSUE TITLE
Manually call refreshCSS or reload

### ISSUE THREAD

**Issue by raphaelokon** [tapio/live-server#220]
Hi there. 

Is there a way to manually call a reload or refresh of the page via the `live-server` API?

Cheers

**Comment by tapio** [comment]
Nope, though it would be quite easy to add as the connected websockets are already saved in a local variable (`clients`). What's your use case?

**Comment by raphaelokon** [comment]
Cheers. I already added the API locally to test—it works pretty good …

```js
	LiveServer.refreshCSS = function() {
		if (clients.length){
			clients.forEach(function(ws) {
				if (ws)
					ws.send('refreshcss');
			});}
	}
	
	LiveServer.reload = function() {
		if (clients.length){
			clients.forEach(function (ws) {
				if (ws)
					ws.send('reload');
			});
		}
	}
```

Our main two use cases with `patternlab-node` regarding manual triggers are: 
1. We have selective watches already built into the `patternlab-node-cli` on which we trigger either only refreshes or reloads manually
2. Make an user facade on which they could add their own watches/actions

/cc @bmuenzenmeyer

**Comment by raphaelokon** [comment]
@tapio Do you think this is something you would merge? Thanks.
